config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  
+  root to: "companies#index"
+
+  resources :companies do
+    resources :jobs
+  end
+  
+  resources :categories
+  resources :jobs
+  resources :comments, only: [:create]
```

Comment:
we already talked in class about how nesting our routes would help us avoid needing to send hidden data 👍